data/examples/Computed+Provision-VU#290915_Coordinator-v2.01.json

```diff
@@ -0,0 +1,396 @@
+{
+  "timestamp": "2020-12-10T18:58:04.153Z",
+  "role": "Coordinator",
+  "id": "VU#290915",
+  "version": "2.0",
+  "computed": "SSVCv2/E:P/V:R/T:P/M:H/D:A/1607626684/",
+  "choices": [
+    {
+      "Exploitation": "poc"
+    },
+    {
+      "Virulence": "rapid"
+    },
+    {
+      "Technical Impact": "partial"
+    },
+    {
+      "Mission &amp; Well-being": "high"
+    },
+    {
+      "Decision": "Attend"
+    }
+  ],
+    "decision_tree": {
+    "decision_points": [
+	{
+	    "label": "Exploitation",
```

Comment:
Should individual decision points also have "descriptions" the way their options do?

Reply:
(If so, please cut a new issue for that so we can merge this and move on.)